packages/collections/lib/Collection.js

```diff
@@ -219,7 +220,37 @@ const clearItems = async (userId, collectionName, { pgdb, loaders }) => {
 const getItemMax = (item) =>
   spreadItemData(item.data && item.data.max ? item.data.max : item)
 
+const PROGRESS_FINISHED_THRESHOLD = 0.75
+const getProgressConditions = (progressState) => {
+  if (!progressState) {
+    return
+  }
+  const aWeekAgo = moment().subtract(7, 'days')
+  const conditions = {
+    FINISHED: {
+      or: [
+        {
+          "data->>'percentage' >=": PROGRESS_FINISHED_THRESHOLD,
+          'updatedAt <=': aWeekAgo,
+        },
+        { "data->>'percentage'": 1 },
+        { "data->>'percentage'": null },
+      ],
+    },
+    UNFINISHED: {
+      or: [
+        { "data->>'percentage' <=": PROGRESS_FINISHED_THRESHOLD },
+        { "data->>'percentage' <": 1, 'updatedAt >': aWeekAgo },
+        { "data->>'percentage'": null },
+      ],
+    },
+  }
+  return conditions[progressState]
+}
```

Comment:
Adds documentation and uses a switch to simplify the code.

```suggestion
const getProgressConditions = (progressState) => {
  if (!progressState) {
    return
  }

  // We consider a recently read artcile only as finished, if it was fully
  // read. An article that has not been looked at for 7 days is considered 
  // read, if more than the threshold (75%) was read.

  const aWeekAgo = moment().subtract(7, 'days')

  switch (progressState) {
    case 'FINISHED':
      return {
        or: [
          {
            "data->>'percentage' >=": PROGRESS_FINISHED_THRESHOLD,
            'updatedAt <=': aWeekAgo,
          },
          { "data->>'percentage'": 1 },
          { "data->>'percentage'": null },
        ],
      }

    case 'UNFINISHED':
      return {
        or: [
          { "data->>'percentage' <=": PROGRESS_FINISHED_THRESHOLD },
          { "data->>'percentage' <": 1, 'updatedAt >': aWeekAgo },
          { "data->>'percentage'": null },
        ],
      }
  }
}
```